shell/assets/translations/en-us.yaml

```
@@ -573,11 +573,21 @@ authConfig:
       label: Endpoints
       custom: Specify
       standard: Generate
+    url: URL
+    realm: Realm
     keycloak:
       url: Keycloak URL
       realm: Keycloak Realm
     issuer: Issuer
     authEndpoint: Auth Endpoint
+    jwksUrl: JWKS URL
+    tokenEndpoint: Token Endpoint
+    userInfoEndpoint: User Info Endpoint
+    acrValue: Authorization Context Reference
+    scope:
+      label: Scope
+      placeholder: openid
+      protip: The scope parameter requires <code>openid</code> to be present. If you are using a custom scope, ensure that openid is included.
```

```suggestion
      protip: The <code>openid</code> scope is required. If you are using custom scopes, ensure that it is included.
```

there's one scope property, but each entry is commonly known as a scope.